examples/dom/WebReconciler.re

```diff
@@ -152,14 +152,14 @@ let reducer = (state, action) =>
   | Decrement => state - 1
   };
 
-let renderCounter = () =>
-  useReducerExperimental(reducer, 0, ((count, dispatch)) =>
-    <view>
-      <button title="Decrement" onPress={() => dispatch(Decrement)} />
-      <text> {"Counter: " ++ str(count)} </text>
-      <button title="Increment" onPress={() => dispatch(Increment)} />
-    </view>
-  );
+let renderCounter = slots => {
+  let ((count, dispatch), _slots) = slots |> useReducer(reducer, 0);
```

Comment:
Thanks for updating the example - made it easy to see the impact of the `slots` model in the API 👍